VoteStakeRegistry/components/Account/LockTokensAccount.tsx

```diff
@@ -362,6 +364,11 @@ const LockTokensAccount = ({ tokenOwnerRecordPk }) => {
         )}
       </div>
       {connected && <Account withHeader={false}></Account>}
+      {realmInfo?.enableNotifications && (
```

Comment:
@SebastianBor  Assuming this is how you wanted to enable/disable notifications feature for a realm.

Reply:
Yes, thank you, however the feature toggle name should be more specific to your product: `enableNotifi`
It's the only notification option atm so I think it's ok for now to have it enabled by default for better discoverability and to give DAOs a chance to try it.
Please change the name and enable it for all realms and it's good to go

Reply:
Pushed an update to address this comment and add the footer for "learn more"